bamboo/bootstrap-integration-tests.sh

```diff
@@ -21,10 +21,8 @@ if [[ $USE_TERRAFORM_ZIPS == true ]]; then
   sed -i "s/source[ ]*= \"..\/..\/tf-modules\/cumulus_ecs_service/source = \"https:\/\/github.com\/nasa\/cumulus\/releases\/download\/$VERSION_FLAG\/terraform-aws-cumulus-ecs-service.zip\/\//g" *.tf
   # Update to use cumulus core module
   sed -i "s/source[ ]*= \"..\/..\/tf-modules\/cumulus/source = \"https:\/\/github.com\/nasa\/cumulus\/releases\/download\/$VERSION_FLAG\/terraform-aws-cumulus.zip\/\/tf-modules\/cumulus/g" *.tf
-
-  ## [MHS, 04/29/2021] fix cumulus_distribution package not released separately.
-  sed -i "s/source[ ]*= \"..\/..\/tf-modules\/cumulus_distribution\"/source = \"https:\/\/github.com\/nasa\/cumulus\/releases\/download\/$VERSION_FLAG\/terraform-aws-cumulus.zip\/\/tf-modules\/cumulus_distribution\"/g" *.tf
-
+  # Update to use distribution module
+  sed -i "s/source[ ]*= \"..\/..\/tf-modules\/distribution/source = \"https:\/\/github.com\/nasa\/cumulus\/releases\/download\/$VERSION_FLAG\/terraform-aws-cumulus-distribution.zip\/\//g" *.tf
```

Comment:
```suggestion
  sed -i "s/source[ ]*= \"..\/..\/tf-modules\/distribution/source = \"https:\/\/github.com\/nasa\/cumulus\/releases\/download\/$VERSION_FLAG\/terraform-aws-cumulus.zip\/\//g" *.tf
```

I'm not confident the rest of the string is correct but we need to be looking at the `terraform-aws-cumulus.zip` and not the distribution zip